tests/frontend/test_kibana_app/cypress/integration/pageobjects/wazuh-menu/wazuh-menu.page.js

```diff
@@ -1,3 +1,74 @@
-export const wazuhButtonSelector = '.eui > .euiFlexGroup';
-export const menuListButtonsSelector = '[class="euiButtonEmpty euiButtonEmpty--text wz-menu-button "]';
-export const subMenuListButtonsSelector = '[class="euiSideNavItem euiSideNavItem--trunk"]';
+export const wazuhButton = '.eui > .euiFlexGroup';
+
+//region Menu
+export const modulesButton = '.wz-menu-sections > :nth-child(1) > .euiButtonContent';
```

Comment:
does this declarations follow any logic?
can we use alpha order? or should we follow the same order in which the elements are on the app?

Reply:
Currently follow the order of the app